### PR TITLE
GitHub hd2605 tressange

### PR DESCRIPTION
**Pull request by hd2605**
### Description of Change

-cloud adoption process

### Test results

- Did you do any localized testing?

in the cloud

/cc: @githubteacher

### CODE REVIEW

File: hd2605 bicycle [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Haibike
```
Comment (hd2605) [inline hd2605 bicycle:1]: good bicycle

File: hd2605 bicycle [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+Electrical
```
Comment (hd2605) [inline hd2605 bicycle:2]: fantastic autonomy

Comment (ThomasHenin) [reply hd2605 bicycle:2]: Great testing 100% completion

**Review by ThomasHenin — COMMENTED** [review]
Good one.

File: hd2605 bicycle (hunk 1 above)
Comment (Claire-CConti) [reply hd2605 bicycle:1]: Tressange is a great place to to bicycle ?

**Review by SteveBaston — APPROVED** [review]

**Review by BernardVillar — APPROVED** [review]
Great!

**Review by ThomasHenin — APPROVED** [review]

**Review by hd2605 — COMMENTED** [review]
final review

**Comment by hd2605** [comment]
time to merge approved by hd

**Pull request merged**